src/modules/media/services/media.service.ts

```diff
@@ -21,6 +21,7 @@ import { DataService } from '~modules/data/services/data.service';
 export class MediaService {
 	private logger: Logger = new Logger(MediaService.name, { timestamp: true });
 	private gotInstance: Got;
+	private referer: string;
```

Comment:
also a referer here, should not be saved in the service to avoid cross request contamination